source/install/ee-prod-rhel-6.rst

```diff
@@ -357,4 +357,4 @@ Test setup and configure Mattermost Server
 8. Feel free to modify other settings
 9. Restart the Mattermost Service by typing:
 
-   -  ``sudo restart mattermost``
+   -  ``sudo service mattermost restart``
```

Comment:
Can you double-check this? RHEL 6 uses Upstart, where the command to start a service is of the form `start $job`